pandas/tests/indexing/test_loc.py

```diff
@@ -888,6 +888,7 @@ def test_identity_slice_returns_new_object(self):
         original_series[:3] = [7, 8, 9]
         assert all(sliced_series[:3] == [7, 8, 9])
 
+    @pytest.mark.xfail(reason="accidental fix reverted - GH37497")
     def test_loc_copy_vs_view(self):
```

Comment:
im inclined to think this test is wrong.

```
orig = DataFrame(zip(range(3), range(3)), columns=["a", "b"])
df = orig.copy()

ser = df.loc[:, "a"]
ser += 2

tm.assert_frame_equal(df, orig)
```

`df.loc[:, "a"]` is a view on `df`, and `+=` is specifically an inplace op (as long as it doesnt result in a dtype change).  I think `df` _should_ be altered.

Reply:
> im inclined to think this test is wrong.

Yep, I had the same feeling: https://github.com/pandas-dev/pandas/pull/36498#issuecomment-719042038